src/main/scala/com/tapad/docker/DockerCommands.scala

```diff
@@ -88,6 +88,13 @@ trait DockerCommands {
    */
   def runVariablesForSubstitutionTask(state: State): Vector[(String, String)] = {
     val extracted = Project.extract(state)
-    extracted.runTask(variablesForSubstitutionTask, state)._2.toVector
+    val (_, value) = extracted.runTask(variablesForSubstitutionTask, state)
+    value.toVector
+  }
+
+  def runVariablesForTestEnvTask(state: State): Map[String, String] = {
```

Comment:
Please move `runVariablesForTestEnvTask` to be inside of `ComposeTestRunner` as this code is specific to test case execution and `DockerCommands` was intended to be for code that calls Docker directly or in the case of `runVariablesForSubstitutionTask` code that gets called which ends up modifying the docker-compose file.

Also, please add a comment to the function for completeness.